packages/e2e/cypress/support/commands.ts

```diff
@@ -0,0 +1,67 @@
+// ***********************************************
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+//
+//
+// -- This is a parent command --
+// Cypress.Commands.add('login', (email, password) => { ... })
+//
+//
+// -- This is a child command --
+// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+import '@testing-library/cypress/add-commands';
+import { cond, constant, eq } from 'lodash/fp';
+
+/**
+ * Using Date.now() for UNKNOWN status gives us a unique and unused
+ * alias
+ */
+const appendStatusToAlias = (status: string) =>
+  `${Cypress.env('USERNAME')}+${status === 'UNKNOWN' ? Date.now() : status}`;
+
+/**
+ * This helper function uses a certain country code matching a test user
+ * with the given status in each applicable environment
+ */
+const countryCodeByStatus = (status: string) => {
+  switch (status) {
+    case 'CONFIRMED':
+      return '1';
+    case 'UNKNOWN':
+      return '2';
+    case 'UNCONFIRMED':
+      return '3';
+  }
+};
```

Comment:
I may revisit these helpers function. Maybe they should live in a separate `helpers.ts` file?

Reply:
I like the idea of having `helpers.ts` file that we can put constants in.

Reply:
If it's specific to Cypress, then having it near to `e2e/cypress` vs. `@aws-amplify/ui` is my only constraint.

But I don't see much value in moving these to yet another file since they're needed for these commands TBH, because there's not re-use.